simplesamlphp_auth.module

```diff
@@ -680,7 +680,7 @@ function _simplesamlphp_auth_generate_block_text() {
     . '<br />' . l(t('Log Out'), 'user/logout') . '</p>';
   }
   else {
-    $block_content .= '<p>' . l(t('Federated Log In'), 'saml_login') . '</p>';
+    $block_content .= '<p>' . l($link_display_name, 'saml_login') . '</p>';
```

Comment:
`$block_content .= '<p>' . config_get('simplesamlphp_auth.settings', 'login_link_display_name') . '</p>';`

Reply:
@laryn , I added and tested all updates and they work great except for this line.  I see link display name correctly, but actual link is missing.  Not sure why this configuration is not picking up saml_login url.  Thank you!!

Reply:
Ha, it's because I gave you wrong info and accidentally left out the link part (the `l` function creates the link). Oops!

Try:  `$block_content .= '<p>' . l(config_get('simplesamlphp_auth.settings', 'login_link_display_name'), 'saml_login') . '</p>';`

Reply:
works!!!